### ISSUE TITLE
Validate should also check enum variant spelling

### ISSUE THREAD

**Issue by infogulch** [devshawn/kafka-gitops#17]
Hi!

While using `apply`, I found that I misspelled the `resource_pattern` field as `PREFIX` when [it should have been `PREFIXED`](https://kafka.apache.org/20/javadoc/org/apache/kafka/common/resource/PatternType.html) and got this error.

```
java.lang.IllegalArgumentException: No enum constant org.apache.kafka.common.resource.PatternType.PREFIX
...
```

I did run the `validate` command beforehand, but it did not catch this error.  I think validate should check enum fields to make sure they are valid before declaring that the state file is ready.

**Comment by devshawn** [comment]
Thanks for the suggestion/report! I'll have this in the next release.

**Comment by devshawn** [comment]
The validate command should support this now, released in [0.2.5](https://github.com/devshawn/kafka-gitops/releases/tag/0.2.5). 😄

**Comment by infogulch** [comment]
Thank you!
